src/app/page.tsx

```diff
@@ -12,7 +12,140 @@ export default function Home() {
             - gap 4 => 0.125rem * 4 = 0.5rem
```

Comment:
not related to this code in PR but in this file page.tsx, login button imported in first line but not used

Reply:
Thanks I removed Login button from the home page when I resolved the merge conflict but didnt think of the import. Will remove in next PR